subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -27,10 +27,34 @@ extern "C" {
  */
 #define STATUS_MAX_LEN (12)
 
-int aws_fota_parse_notify_next_document(char *job_document,
-		u32_t payload_len, char *job_id_buf, char *hostname_buf,
-		char *file_path_buf);
+/**
+ * @brief Parse a given AWS IoT Job Execution JSON object.
+ *
+ * @param[in] job_document JSON formated string of an AWS Job execution
+ * @param[in] payload_len The length of the provided string.
+ * @param[out] job_id_buf Output buffer for the Job ID string read from the JSON
+ * @param[out] hostname_buf Output buffer for the hostname read from the JSON
+ * @param[out] file_path_buf Output buffer for the file path read from the JSON
+ * @param[out] version_number Job document version number.
+ *
+ * @return 1 if the Job Execution object is empty, 0 if parsed correctly,
```

Comment:
```suggestion
 * @return 1 if job_document is empty, 0 if parsed correctly,
```

Reply:
No it returns 1 if it is able to decode timestamp field and clientToken field only